exchanges/gateio/gateio_test.go

```diff
@@ -2544,14 +2537,10 @@ func TestUpdateOrderExecutionLimits(t *testing.T) {
 	testexch.UpdatePairsOnce(t, g)
 
 	err := g.UpdateOrderExecutionLimits(t.Context(), 1336)
-	if !errors.Is(err, asset.ErrNotSupported) {
-		t.Fatalf("received %v, expected %v", err, asset.ErrNotSupported)
-	}
+	require.ErrorIs(t, err, asset.ErrNotSupported)
 
 	err = g.UpdateOrderExecutionLimits(t.Context(), asset.Options)
-	if !errors.Is(err, common.ErrNotYetImplemented) {
-		t.Fatalf("received %v, expected %v", err, common.ErrNotYetImplemented)
-	}
+	require.ErrorIs(t, err, common.ErrNotYetImplemented)
```

Comment:
Flagging future snowstorm update for cases below:  `err != nil { t.Fatal(err) }`

Reply:
This is coming hahaha